kmath-ast/src/jvmMain/kotlin/scientifik/kmath/asm/AsmExpressions.kt

```diff
@@ -0,0 +1,247 @@
+package scientifik.kmath.asm
+
+import scientifik.kmath.expressions.Expression
+import scientifik.kmath.expressions.ExpressionAlgebra
+import scientifik.kmath.operations.*
+import kotlin.reflect.full.memberFunctions
+import kotlin.reflect.jvm.jvmName
+
+interface AsmExpression<T> {
```

Comment:
Add documentation since the interface is public.

Reply:
OK